src/styles/card.css

```diff
@@ -0,0 +1,116 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 16px;
+  width: 200px;
+  color: var(--main);
+}
+
+.product-card__img {
+  display: block;
+  height: 134px;
+  width: 160px;
+  margin: 16px auto 40px;
+}
+
+.product-card__header {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin-bottom: 4px;
+}
+
+.product-card__code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--secondary);
+}
+
+.product-card__review {
+  display: flex;
+  align-items: flex-start;
+  justify-content: space-between;
+  margin-top: 16px;
+  margin-bottom: 24px;
+}
+
+.product-card__feed {
+  font-size: 10px;
+  align-self: center;
+  line-height: 14px;
+}
+
+.stars {
+  display: flex;
+  justify-content: flex-start;
+}
+
+.star {
+  height: 16px;
+  width: 16px;
+  background: url(../images/star.svg) no-repeat center;
+}
+
+.star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--4 :not(:last-child) {
+  background: url(../images/star-active.svg) no-repeat center;
+}
```

Comment:
You should reuse stars block from previous task

Reply:
It`s not OK, I didn`t use entire 0 to 5th block of stars selector but only 4th?

Reply:
Yes, it is in the task: Reuse stars block from [Stars task](https://github.com/mate-academy/layout_stars) and keep it operational. If you change stars--4 modifier to stars--2 or other - number of selected stars should change correctly.